pytorch_lightning/callbacks/device_stats_monitor.py

```diff
@@ -0,0 +1,71 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
let's include the file in the mypy checklist in pyproject.toml :)

Reply:
sure. is there a reason we don't run mypy on all files rather than having to add them individually?

Reply:
because lightning is not fully typed, it's a large effort to make it mypy compatible: #7037 #5023
we just want to make sure newly added code files are already type checked, otherwise we will never reach the goal. 
see also the mypy ci job

Reply:
got it, thanks for explaining! makes sense